### DISCUSSION TITLE
No such file or directory: 'requirements.txt';  Could not find a version that satisfies the requirement syft (unavailable)

### DISCUSSION THREAD

**Discussion by Spartan-119** [OpenMined/courses#293] [Course: Foundations of Private Computation] [answered]
I created a virtual environment called '_privateai_'. 
**NOTE:** _I am using Anaconda prompt_
Then I cloned the GitHub repo to get the exercise notebooks (using git bash).
After that as per the instructions I ran the command to install requirements.txt. 
**Got an error**.
Then I ran the "pip install git+https://github.com/OpenMined/PySyft@dev#egg=syft" command. **Got the error** as shown in the image.
![error](https://user-images.githubusercontent.com/36173893/111948278-ee5af300-8b04-11eb-92e4-5c3b85431ba6.PNG)

**Comment by znreza** [comment, marked as answer]
@Spartan-119 the previous error for "pip install -r requirements.txt" says that "No such file or directory". That's because you are running from outside of PySyft folder where it cannot locate the requirements.txt file. Please run it from inside of PySyft folder or give the absolute path of of the file. After that, try installing syft again from PySyft folder. It should work.

**Reply by Spartan-119** [reply]
@znreza Thank you. Your solution helped, and it is working fine now :)
